easybuild/easyconfigs/d/DeepCell/DeepCell-0.11.1-foss-2021b-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,64 @@
+easyblock = 'PythonBundle'
+
+name = 'DeepCell'
+version = '0.11.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://deepcell.readthedocs.io/en/master/'
+description = """deepcell-tf is a deep learning library for single-cell analysis of biological
+ images.This library allows users to apply pre-existing models to imaging data as well as to
+ develop new deep learning models for single-cell analysis."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('IPython', '7.26.0'),
+    ('OpenCV', '4.5.5', '-contrib'),
+    ('tqdm', '4.62.3'),
+    ('matplotlib', '3.4.3'),
+    ('scikit-learn', '1.0.1'),
+    ('scikit-image', '0.19.1'),
+    ('CUDA', '11.4.1', '', True),
+    ('TensorFlow', '2.7.1', versionsuffix),
+    ('spektral', '1.1.0', versionsuffix),
+    ('Graphviz', '2.50.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('pydot', '1.4.2', {
+        'checksums': ['248081a39bcb56784deb018977e428605c1c758f10897a339fce1dd728ff007d'],
+    }),
+    ('typeguard', '2.13.3', {
+        'checksums': ['00edaa8da3a133674796cf5ea87d9f4b4c367d77476e185e80251cc13dfbb8c4'],
+    }),
+    ('tensorflow-addons', '0.17.0', {
+        'source_tmpl': 'tensorflow_addons-%(version)s-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl',
+        'unpack_sources': False,
+        'checksums': ['becea12c19e007831195c30aaba97ad51b98b67fe9083d1e2e8375beab8d7401'],
+    }),
+    ('DeepCell-Toolbox', '0.11.2', {
+        'patches': ['DeepCell_Toolbox-0.11.2_deps.patch'],
```

Comment:
@fizwit please include a comment explaining the need for the patch, either here or in the patch itself